README.md

```diff
@@ -51,18 +51,18 @@ daily:20.10                 devel,groovy      20200721         Ubuntu 20.10
 
 ## Launch a fresh instance of the current Ubuntu LTS
 ```
-$ multipass launch ubuntu
+$ multipass launch 20.04
```

Comment:
Since the heading of section specifies launching the current Ubuntu LTS, let's change this to `multipass launch lts` instead.